src/types/hashable/Unspent.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+from src.types.hashable.Coin import Coin
+from src.types.sized_bytes import bytes32
+from src.util.streamable import Streamable, streamable
+from src.util.ints import uint32, uint8
+
+
+@dataclass(frozen=True)
+@streamable
+class Unspent(Streamable):
```

Comment:
I don't really like the name Unspent, it caused confusion for me. Since it can either be spent or not. Maybe call it CoinRecord or CoinSpend?

Reply:
CoinRecord sounds appropriate